airflow/auth/managers/fab/fab_auth_manager.py

```diff
@@ -39,3 +39,7 @@ def get_user_name(self) -> str:
         first_name = current_user.first_name or ""
         last_name = current_user.last_name or ""
         return f"{first_name} {last_name}".strip()
+
+    def is_logged_in(self) -> bool:
+        """Return whether the user is logged in."""
+        return not current_user.is_anonymous
```

Comment:
should we check if `current_user` is None before ? Given the fact that it was done in https://github.com/apache/airflow/pull/32428/files#diff-8075dbc4ae06e7b83cd163eeb206322a7977c227bb57398e7343c9e09c80ae59L71

Reply:
I am not sure it can happen but I dont mind adding it